pkg/topology/kademlia/kademlia.go

```diff
@@ -1166,3 +1173,21 @@ func (k *Kad) Close() error {
 
 	return nil
 }
+
+func randomSubset(addrs []swarm.Address, count int) ([]swarm.Address, error) {
+
+	if count >= len(addrs) {
```

Comment:
The addresses aren't shuffled when the count `count >= len(addrs)`, is this expected behavior?

Reply:
yes